kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -16,15 +16,77 @@ This example shows
 	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
 
+* Diagram Properties (top)
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties (bottom)
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties (process variables)
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* Start Message
+<p align="center"><img src="docs/images/startMessage.png"></p>
+
+* Start Message (Assignments)
+<p align="center"><img src="docs/images/startMessageAssignments.png"></p>
+
+* Process Traveler Business Rule (top)
+<p align="center"><img src="docs/images/processTravelerBusinessRule.png"></p>
+
+* Process Traveler Business Rule (bottom)
+<p align="center"><img src="docs/images/processTravelerBusinessRule2.png"></p>
+
+* Process Traveler Business Rule (Assignments)
+<p align="center"><img src="docs/images/processTravelerBusinessRuleAssignments.png"></p>
+
+* Process Traveler Gateway
+<p align="center"><img src="docs/images/processedTravelerGateway.png"></p>
+
+* Process Traveler Gateway Yes Connector
+<p align="center"><img src="docs/images/processedTravelerYesConnector.png"></p>
+
+* Process Traveler Gateway No Connector
+<p align="center"><img src="docs/images/processedTravelerNoConnector.png"></p>
+
+* Log Traveler Script Task
+<p align="center"><img src="docs/images/logTravelerScriptTask.png"></p>
+
+* Skip Traveler Script Task
+<p align="center"><img src="docs/images/skipTravelerScriptTask.png"></p>
+
+* Processed Traveler End Message
+<p align="center"><img src="docs/images/processedTravelerEndMessage.png"></p>
+
+* Processed Traveler End Message (Assignments)
+<p align="center"><img src="docs/images/processedTravelerEndMessageAssignments.png"></p>
+
+* Skip Traveler End
+<p align="center"><img src="docs/images/skipTraveler.png"></p>
+
+
 ## Infrastructure requirements
 
 This quickstart requires an Apache Kafka to be available and by default expects it to be on default port and localhost.
 
+* Install and Startup Kafka Server / Zookeeper
+
+```
+https://kafka.apache.org/downloads
```

Comment:
only link to quickstart is enough. No need for the downloads link as it is already pointed to in quickstart

Reply:
fixed @radtriste